### PR TITLE
data: VaradBelwalkar

### PR DESCRIPTION
**Pull request by VaradBelwalkar**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5915"></a>

### CODE REVIEW

File: data/VaradBelwalkar.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+	"name": "Varad Belwalkar",
+	"displayStatsPublic": true,
+	"type": "personal",
+	"bio": "DevOps and Machine Learning Enthusiast",
+	"avatar": "https://github.com/VaradBelwalkar.png",
```
Comment (stevenovak123) [inline data/VaradBelwalkar.json:6]: `avatar` is not a required field anymore

Comment (stevenovak123) [reply data/VaradBelwalkar.json:6]: https://linkfree.io/docs/full-profile-example. Check the example json file here and make changes. Also can go on the playground https://linkfree.io/playground to verify if you have a valid json after making changes

**Review by stevenovak123 — CHANGES_REQUESTED** [review]

File: data/VaradBelwalkar.json (hunk 1 above)
Comment (VaradBelwalkar) [reply data/VaradBelwalkar.json:6]: Great! will make those changes!

**Review by shubhsharma19 — APPROVED** [review]
LGTM 👍

**Review by shubhsharma19 — APPROVED** [review]

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @VaradBelwalkar, thank you for contributing :tada: 
Please close the duplicate PR #5914

**Review by vedprakashnautiyal — APPROVED** [review]

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**